### DISCUSSION TITLE
Change Netbox Logo

### DISCUSSION THREAD

**Discussion by amity2002** [netbox-community/netbox#17750] [Help Wanted!]
When I was on versions prior to 4.1.0 I changed the logo in /opt/netbox/netbox/static (files changed: netbox_logo.svg, netbox_icon.svg)
Now when I'm in version 4.1.0 the logos have changed to Netbox logos.
Is there a new feature to display logos?

Thanks for your help

**Comment by candlerb** [comment]
In your browser, right-click on the image and "copy image address" or "open image in new tab" to find the URL.

For example, on the homepage of Netbox 4.1.3, if I do this for the very top-left logo ("Netbox community") it shows me `/logo_netbox_dark_teal.svg`

Someone wrote a plugin to change the UI colours, and they've also documented how to change the logos:
https://github.com/PaulR282/netbox_recolor

**Comment by marsteel** [comment]
Logo SVG are copied from project-static/img/logo_netbox_bright_teal.svg and logo_netbox_dark_teal.svg to static/ path each time 
upgrade.sh runs or the container starts if you use container to deploy NetBox 

Replace the files with your logo files and automate that action as part of upgrade process.

Logo will be displayed in Login page and top left in the navigation menu. You might need to change the CSS file in case your Logo SVG size is different with the original NetBox logo file otherwise the logo size is okay in Login page but too small in the navigation menu.

```
sed -i -e 's/.navbar-brand-image{height:2rem/.navbar-brand-image{height:6rem/g' /opt/netbox/netbox/project-static/dist/netbox.css
sed -i -e 's/.navbar-brand-image{height:2rem/.navbar-brand-image{height:6rem/g' /opt/netbox/netbox/static/netbox.css
```
